packages/web/src/utils/validators.ts

```diff
@@ -0,0 +1,256 @@
+import { BigNumber } from 'bignumber.js'
+import { ValidatorsListProps } from 'src/dev/ValidatorsList'
+import { styles } from 'src/dev/ValidatorsListStyles'
+import { weiToDecimal } from 'src/utils/utils'
+
+export const localStoragePinnedKey = 'pinnedValidators'
+
+export interface Edges<T> {
+  edges: Array<{
+    node: T
+  }>
+}
+
+export type Address = string
+
+export interface CeloValidatorGroup {
+  account: {
+    address: string
+    lockedGold: string
+    name: string
+    usd: string
+    claims: Edges<{
+      verified: boolean
+      element: string
+    }>
+  }
+  accumulatedActive: string
+  accumulatedRewards: string
+  affiliates: Edges<{
+    account: {
+      claims: Edges<{
+        verified: boolean
+        element: string
+      }>
+    }
+    address: string
+    attestationsFulfilled: number
+    attestationsRequested: number
+    lastElected: number
+    neverElected: boolean
+    lastOnline: number
+    lockedGold: string
+    name: string
+    score: string
+    usd: string
+  }>
+  commission: string
+  numMembers: number
+  receivableVotes: string
+  rewardsRatio: string
+  votes: string
+}
+
+export interface CeloGroup {
+  id: number
+  elected: number
+  online: number
+  total: number
+  uptime: number
+  attestation: number
+  name: string
+  address: Address
+  usd: number
+  celo: number
+  receivableRaw: number
+  receivableVotes: string
+  votesRaw: number
+  votes: string
+  votesAbsolute: string
+  commission: number
+  rewards: number
+  rewardsStyle: any
+  numMembers: number
+  claims: string[]
+  validators: Array<{
+    name: string
+    address: Address
+    usd: number
+    celo: number
+    elected: boolean
+    neverElected: boolean
+    online: boolean
+    uptime: number
+    attestation: number
+    claims: string[]
+  }>
+}
+
+export const orderAccessors = {
+  order: (_) => _.order,
+  name: (_) => (_.name || '').toLowerCase() || null,
+  total: (_) => _.numMembers * 1000 + _.elected,
+  votes: (_) => +_.votesAbsolute || 0,
+  rawVotes: (_) => _.votesRaw || 0,
+  votesAvailables: (_) => _.receivableRaw || 0,
+  celo: (_) => _.celo || 0,
+  commission: (_) => _.commission || 0,
+  rewards: (_) => _.rewards || 0,
+  uptime: (_) => _.uptime || 0,
+  attestation: (_) => _.attestation || 0,
+}
+
+/**
+ * Determines whether a validator is pinned
+ * Pinned validators are set to the top of the table, regardless of how the table is sorted
+ * @param  {Address}    address The address of a given Validator
+ * @return {boolean}    Whether the given validator is pinned
+ */
+export function isPinned(address: Address) {
+  const list = (localStorage.getItem(localStoragePinnedKey) || '').split(',') || []
+  return +list.includes(address)
```

Comment:
`+` coercese to a number so return will be 1 or 0? i think (not boolean like return doc says

Reply:
Yes, good catch